in-cluster/default.yaml

```diff
@@ -468,3 +468,15 @@ spec:
               message: "There are PVCs using more than 80% of storage: {{ .PVC.ConcatenatedNames }}"
           - pass:
               message: "No PVCs are using more than 80% of storage"
+    - event:
+        checkName: event-oom-check
+        namespace: default
+        reason: "OOMKilling"
+        kind: Node
+        outcomes:
+          - fail:
+              when: "true"
+              message: Event {{ .Reason }} by object {{ .InvolvedObject.Name }} kind {{ .InvolvedObject.Kind }} has message {{ .Message }}
```

Comment:
nit: Perhaps we can have a message like below

```
Pod '{{ .InvolvedObject.Name }}' got killed (OOMKilling) with the reason: {{ .Message }}
```

Reply:
given that this event is kind: Node, is the object name not the node name?  I guess we need an example event to confirm.

Reply:
Good point.

Reply:
@5hubh49 perhaps a good follow-up would be detecting which pod. Something like below. I tried to get a pod being OOMKilled, and emit an event, but couldn't

```yaml
    - event:
        checkName: OOMkilled detector
        namespace: default
        reason: "OOMKilled"
        kind: Pod
        outcomes:
          - fail:
              when: "true"
              message: Pod '{{ .InvolvedObject.Name }}' got killed with reason: {{ .Message }}
          - pass:
              when: "false" 
              message: No OOM killed pods found
```